ui/overflow_menu.js

```diff
@@ -230,6 +230,13 @@ shaka.ui.OverflowMenu = class extends shaka.ui.Element {
         this.localization.resolve(LocIds.MORE_SETTINGS);
   }
 
+  /**
+   * @private
+   */
+  computeOverFlowMenuPos_() {
+    this.computeMaxHeight_();
+    this.computeHorizontalPosition_();
```

Comment:
Merge these two methods into a single one, thus avoiding duplicate calls to getBoundingClientRect

Reply:
I separated this into two functions because they do separate things. But yes we can combine them.